be/src/common/config.h

```diff
@@ -553,9 +553,10 @@ CONF_String(default_rowset_type, "BETA");
 CONF_Int64(brpc_max_body_size, "3147483648");
 // Max unwritten bytes in each socket, if the limit is reached, Socket.Write fails with EOVERCROWDED
 CONF_Int64(brpc_socket_max_unwritten_bytes, "1073741824");
-// Whether to transfer RowBatch in ProtoBuf Request to Controller Attachment and send it
-// through brpc, this will be faster and avoid the error of Request length overflow.
-CONF_mBool(transfer_data_by_brpc_attachment, "false");
+// Whether to embed the ProtoBuf Request serialized string together with Tuple/Block data into
+// Controller Attachment and send it through http brpc when the length of the Tuple/Block data
+// is greater than 1.8G. This is to avoid the error of Request length overflow (2G).
+CONF_mBool(brpc_request_embed_attachment_send_by_http, "true");
```

Comment:
maybe `transfer_large_data_by_brpc`

Reply:
The default value is true. It is compatible that an new version BE send request to a old version BE?

Reply:
> transfer_large_data_by_brpc

done

Reply:
> The default value is true. It is compatible that an new version BE send request to a old version BE?

yep, should be false, we can set true in v1.2